packages/react/src/css/index.js.flow

```diff
@@ -1,35 +1,36 @@
 /**
  * Flowtype definitions for index
  * Generated by Flowgen from a Typescript Definition
- * Flowgen v1.15.0
+ * Flowgen v1.17.0
  * @flow
  */
-import type { BasicTemplateInterpolations, CSSProps, FunctionInterpolation } from '../types';
+import type { CSSProps, CssObject, CssFunction } from '../types';
 /**
- * Create styles that can be re-used between components with a template literal.
+ * ## CSS
  *
- * ```
- * css`color: red;`;
- * ```
+ * Define styles that are statically typed and useable with other Compiled APIs.
+ * For further details [read the documentation](https://compiledcssinjs.com/docs/api-css).
  *
- * For more help, read the docs:
- * https://compiledcssinjs.com/docs/api-css
- * @param css
- * @param values
- */
-declare export default function css<T>(
-  _css: $ReadOnlyArray<string>,
-  ..._values: (BasicTemplateInterpolations | FunctionInterpolation<T>)[]
-): CSSProps;
-/**
- * Create styles that can be re-used between components with an object
+ * ### Style with objects
+ * @example ```
+ * const redText = css({
+ *   color: 'red',
+ * });
  *
+ * <div css={redText} />
  * ```
- * css({ color: 'red' });
- * ```
  *
- * For more help, read the docs:
- * https://compiledcssinjs.com/docs/api-css
- * @param css
+ * ### Style with template literals
+ * @example ```
+ * const redText = css`
+ *   color: red;
+ * `;
+ *
+ * <div css={redText} />
+ * ```
  */
-declare export default function css(_css: CSSProps): CSSProps;
+declare export default function css<TProps>(
```

Comment:
Doesn't flow complain when you export two overloaded functions?

Reply:
seems to work fine from my testing - I'll double-check while fixing the other issues